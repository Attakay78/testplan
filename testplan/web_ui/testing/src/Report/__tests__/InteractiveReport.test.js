/* Tests the InteractiveReport component. */
import React from 'react';
import {shallow} from 'enzyme';

import InteractiveReport from '../InteractiveReport';

describe('InteractiveReport', () => {
  it("shallow renders and matches snapshot", () => {
    const interactiveReport = shallow(<InteractiveReport />);
    expect(interactiveReport).toMatchSnapshot();
  });
});

