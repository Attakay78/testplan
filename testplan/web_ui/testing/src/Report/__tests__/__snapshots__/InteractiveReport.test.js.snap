// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InteractiveReport shallow renders and matches snapshot 1`] = `
<EmptyReport
  message="Interactive mode not yet implemented..."
/>
`;
